x-pack/docs/en/rest-api/security/saml-authenticate-api.asciidoc

```diff
@@ -0,0 +1,86 @@
+[role="xpack"]
+[[security-api-saml-authenticate]]
+=== SAML authenticate API
+
+Submits a SAML response message to {es} for consumption.
+
+[[security-api-saml-authenticate-request]]
+==== {api-request-title}
+
+`POST /_security/saml/authenticate`
+
+[[security-api-saml-authenticate-desc]]
+==== {api-description-title}
+
+The SAML response can be a response to a SAML authentication request that was
+previously created using the
+<<security-api-saml-prepare-authentication, SAML prepare authentication API>>.
```

Comment:
I find this sentence hard to parse (too many request / response terms, I think).
I don't have a suggestion right now, but since it's the first sentence in the body, it's probably worth trying to make it clear.

Reply:
I have no good alternatives either :/ The fact that we have two pairs of requests/responses we need to talk about here, doesn't help. I broke down the sentence a bit hoping this makes it clearer but I'm open to suggestions